src/Reflection/PhpDefect/PhpDefectClassReflectionExtension.php

```diff
@@ -55,65 +55,65 @@ class PhpDefectClassReflectionExtension implements PropertiesClassReflectionExte
 			'xmlStandalone' => 'bool',
 			'xmlVersion' => 'string',
 		],
-		\DOMDocumentType::class => [ // extends DOMNode
+		'DOMDocumentType' => [ // extends DOMNode
 			'publicId' => 'string',
 			'systemId' => 'string',
 			'name' => 'string',
-			'entities' => \DOMNamedNodeMap::class,
-			'notations' => \DOMNamedNodeMap::class,
+			'entities' => 'DOMNamedNodeMap',
+			'notations' => 'DOMNamedNodeMap',
 			'internalSubset' => 'string',
 		],
-		\DOMElement::class => [ // extends DOMNode
+		'DOMElement' => [ // extends DOMNode
 			'schemaTypeInfo' => 'bool',
 			'tagName' => 'string',
 		],
-		\DOMEntity::class => [ // extends DOMNode
+		'DOMEntity' => [ // extends DOMNode
```

Comment:
It might be possible to remove some `ext-*` from composer.json after this change?

Reply:
That depends. All of these are in `require-dev`, and we do use symbols from them in tests. We could replace those symbols with strings as well, but maybe for tests it's better if we keep it like it is? What do you think?

Reply:
OK, we can leave them there 😊

Reply:
I don't understand this change. Referencing unknown class is completely legit in PHP, so why moving back to pre-PHP5.5 and use strings?